Collision/src/main/java/dk/sdu/mmmi/modulemon/Collision/CollisionProcessing.java

```diff
@@ -20,7 +20,7 @@ public class CollisionProcessing implements IPostEntityProcessingService {
 
     @Override
     public void process(GameData gameData, World world) {
-        if(mapView == null){
+        if(mapView == null || gameData.isPaused()){
```

Comment:
Nice that it stops when paused.
But it then oocured to me that the NPCControlSystem isn't stopped when `isPaused()` is true.
So could you either add:

```java
if(gamedata.isPaused()){
    return;
}
```
In `NPCControlSystem.java`. Or maybe even better: Check in Mapviews `update()` method if the game is paused.. If it is, don't process entityProcessors and postEntityProcessors.

Reply:
I've updated the `update` method in MapView to not run entity processors and postentity processors when the game is paused.